src/components/Chart/Bars.js

```diff
@@ -602,6 +580,34 @@ const BarChart = props => {
                       .map((segment, i) => (
                         <segment.Component key={`seg${i}`} />
                       ))}
+                    {showBarValues && (
+                      <>
+                        {bar.min && (
```

Comment:
I find `min` and `max` a bit confusing. Since the prop is called `showBarValues` how about `negValue`and `posValue`instead?

Reply:
Thanks for the feedback: I iterated on the name and ended up with `sumNegative` and `sumPositive`: https://github.com/orbiting/styleguide/pull/394/commits/6c4120f8e9485004e8801315e091590f1330c6a6